Package.swift

```diff
@@ -11,20 +11,9 @@ let package = Package(
     products: [
         .library(name: "SplunkOtel", targets: ["SplunkOtel"])
     ],
-    dependencies: [
-        .package(name: "opentelemetry-swift", url:"https://github.com/open-telemetry/opentelemetry-swift", from: "1.1.4"),
```

Comment:
When using Swift's PM we'd need to modify our codebase to add OpenTelemetry imports (which are removed in the current iteration as they are in the same package), not sure if there'd a better way to do it.

Reply:
I had the same thought as John. I was wondering if there is a way to keep the SPM deployment unaltered and only do this for those that need XCFramework support. It does seem like it would require automation as changing this file, bring the Otel files in, making sure imports are cleaned up, and everything is still building every time would be unfortunate to do manually.

Reply:
Could be doable in the case of XCFramework I suppose, but supporting CocoaPods might be a tad more complicated like this, I need to understand CocoaPods a bit more for this though.

With the source embedding both XCFramework and CocoaPods support _should_ (🤞 ) be trivial